includes/templates/frontend/single-web-story.php

```diff
@@ -33,5 +33,7 @@
 if ( $current_post instanceof WP_Post ) {
 	echo '<!DOCTYPE html>';
 	$renderer = new Story_Renderer( $current_post );
-	echo $renderer->render(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	$rendered = $renderer->render();
+	$rendered = str_replace( ' standalone="standalone"', ' standalone', $rendered ); // @todo Remove! Temporary workaround for AMP plugin validation issue.
+	echo $rendered; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
This change can be reverted now that https://github.com/ampproject/amp-wp/pull/5157 has been merged. Well, that is if `version_compare( AMP__VERSION, '2.0', '>=' )`.

Probably still best to replace `standalone="standalone"` with just `standalone` in the underlying `render` method.

Reply:
I'll update the JS code base to make sure we just print `standalone` in the first place